tests/Aspire.EndToEnd.Tests/Directory.Build.props

```diff
@@ -2,8 +2,11 @@
   <PropertyGroup>
     <IsHelixOnlyTestProject>true</IsHelixOnlyTestProject>
     <IsWorkloadTestProject>true</IsWorkloadTestProject>
-    <RunSettingsFilePath>$(MSBuildThisFileDirectory).runsettings</RunSettingsFilePath>
   </PropertyGroup>
 
   <Import Project="..\Directory.Build.props" />
+
+  <PropertyGroup>
+    <RunSettingsFilePath>$(MSBuildThisFileDirectory).runsettings</RunSettingsFilePath>
```

Comment:
It's because of the properties at the top:
```xml
  <PropertyGroup>
    <IsWorkloadTestProject>true</IsWorkloadTestProject>
  </PropertyGroup>
```

.. which we want to set before `tests/Directory.Build.props` so it imports the workload testing targets.